ledger/catchpointtracker.go

```diff
@@ -270,7 +278,7 @@ func (ct *catchpointTracker) finishFirstStage(ctx context.Context, dbRound basic
 
 // Possibly finish generating first stage catchpoint db record and data file after
 // a crash.
-func (ct *catchpointTracker) finishFirstStageAfterCrash(dbRound basics.Round) error {
+func (ct *catchpointTracker) finishFirstStageAfterCrash(dbRound basics.Round, blockProto protocol.ConsensusVersion) error {
```

Comment:
Here, instead of taking `blockProto`, will be simpler to just have a `bool` variable: `dbRoundEnabledCPWtSPContexts`.
This way, one does not need to be guessing what information is needed from the protocol, and will be clear that this information is based on the `dbRound`.

If you prefer to keep `blockProto`, that is fine, but please rename it to `dbRroundProto` instead, so that it will be clear that it is linked to `dbRound`.

Reply:
I think I agree with this, its not the current consensus version but the one at the time of this block.

Reply:
No what, blockProto is enough for me thinking about this more.